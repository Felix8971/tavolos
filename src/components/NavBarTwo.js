import React from 'react';

class NavBarTwo extends React.Component {
  constructor(props) {
    super();
  }
  
  render() {
    return (
      <nav className='nav-bar-two'>
        <div className='nav-wide'>
          
          <div className="menu">   
            <div className='legend' href="#">Bangkok / </div>
            <a className='link' href="#">Chinese New Year</a>
            <a className='link' href="#">Date Night</a>
            <a className='link' href="#">Group Dinner</a>
            <a className='link' href="#">Buisness Meals</a>
          </div>

          <div className='search'>
            <span className='legend'>Search:</span>
            <a className='link' href="#">Restaurant/Cuisine/Area</a>
          </div>
        </div>

      </nav>
    )
  }
}

export default NavBarTwo;


