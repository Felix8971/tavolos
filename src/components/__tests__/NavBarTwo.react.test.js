import React from 'react';
import { shallow, mount, render } from 'enzyme';
import NavBarTwo from '../NavBarTwo';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
enzyme.configure({ adapter: new Adapter() });

describe('NavBarTwo', () => {

    it('should be defined', () => {
      expect(NavBarTwo).toBeDefined();
    });

    it('should render correctly', () => {
      const tree = shallow(
        <NavBarTwo/>
      );
      expect(tree).toMatchSnapshot();
    });

    it('should contain 5 link', () => {
      const tree = shallow(
        <NavBarTwo/>
      );
      expect(tree.find('a')).toHaveLength(5);
    })

});