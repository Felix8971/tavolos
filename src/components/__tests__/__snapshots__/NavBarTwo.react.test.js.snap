// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBarTwo should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBarTwo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="nav-wide"
      >
        <ul
          className="menu"
        >
          <li>
            <div
              className="legend"
              href="#"
            >
              Bangkok / 
            </div>
          </li>
          <li>
            <a
              className="link"
              href="#"
            >
              Chinese New Year
            </a>
          </li>
          <li>
            <a
              className="link"
              href="#"
            >
              Date Night
            </a>
          </li>
          <li>
            <a
              className="link"
              href="#"
            >
              Group Dinner
            </a>
          </li>
          <li>
            <a
              className="link"
              href="#"
            >
              Buisness Meals
            </a>
          </li>
        </ul>
        <div
          className="search"
        >
          <span
            className="legend"
          >
            Search:
          </span>
          <a
            className="link"
            href="#"
          >
            Restaurant/Cuisine/Area
          </a>
        </div>
      </div>,
      "className": "nav-bar-two",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <ul
            className="menu"
          >
            <li>
              <div
                className="legend"
                href="#"
              >
                Bangkok / 
              </div>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Chinese New Year
              </a>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Date Night
              </a>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Group Dinner
              </a>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Buisness Meals
              </a>
            </li>
          </ul>,
          <div
            className="search"
          >
            <span
              className="legend"
            >
              Search:
            </span>
            <a
              className="link"
              href="#"
            >
              Restaurant/Cuisine/Area
            </a>
          </div>,
        ],
        "className": "nav-wide",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                <div
                  className="legend"
                  href="#"
                >
                  Bangkok / 
                </div>
              </li>,
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Chinese New Year
                </a>
              </li>,
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Date Night
                </a>
              </li>,
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Group Dinner
                </a>
              </li>,
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Buisness Meals
                </a>
              </li>,
            ],
            "className": "menu",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="legend"
                  href="#"
                >
                  Bangkok / 
                </div>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Bangkok / ",
                  "className": "legend",
                  "href": "#",
                },
                "ref": null,
                "rendered": "Bangkok / ",
                "type": "div",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="link"
                  href="#"
                >
                  Chinese New Year
                </a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Chinese New Year",
                  "className": "link",
                  "href": "#",
                },
                "ref": null,
                "rendered": "Chinese New Year",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="link"
                  href="#"
                >
                  Date Night
                </a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Date Night",
                  "className": "link",
                  "href": "#",
                },
                "ref": null,
                "rendered": "Date Night",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="link"
                  href="#"
                >
                  Group Dinner
                </a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Group Dinner",
                  "className": "link",
                  "href": "#",
                },
                "ref": null,
                "rendered": "Group Dinner",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="link"
                  href="#"
                >
                  Buisness Meals
                </a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Buisness Meals",
                  "className": "link",
                  "href": "#",
                },
                "ref": null,
                "rendered": "Buisness Meals",
                "type": "a",
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                className="legend"
              >
                Search:
              </span>,
              <a
                className="link"
                href="#"
              >
                Restaurant/Cuisine/Area
              </a>,
            ],
            "className": "search",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Search:",
                "className": "legend",
              },
              "ref": null,
              "rendered": "Search:",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Restaurant/Cuisine/Area",
                "className": "link",
                "href": "#",
              },
              "ref": null,
              "rendered": "Restaurant/Cuisine/Area",
              "type": "a",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="nav-wide"
        >
          <ul
            className="menu"
          >
            <li>
              <div
                className="legend"
                href="#"
              >
                Bangkok / 
              </div>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Chinese New Year
              </a>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Date Night
              </a>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Group Dinner
              </a>
            </li>
            <li>
              <a
                className="link"
                href="#"
              >
                Buisness Meals
              </a>
            </li>
          </ul>
          <div
            className="search"
          >
            <span
              className="legend"
            >
              Search:
            </span>
            <a
              className="link"
              href="#"
            >
              Restaurant/Cuisine/Area
            </a>
          </div>
        </div>,
        "className": "nav-bar-two",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <ul
              className="menu"
            >
              <li>
                <div
                  className="legend"
                  href="#"
                >
                  Bangkok / 
                </div>
              </li>
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Chinese New Year
                </a>
              </li>
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Date Night
                </a>
              </li>
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Group Dinner
                </a>
              </li>
              <li>
                <a
                  className="link"
                  href="#"
                >
                  Buisness Meals
                </a>
              </li>
            </ul>,
            <div
              className="search"
            >
              <span
                className="legend"
              >
                Search:
              </span>
              <a
                className="link"
                href="#"
              >
                Restaurant/Cuisine/Area
              </a>
            </div>,
          ],
          "className": "nav-wide",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <div
                    className="legend"
                    href="#"
                  >
                    Bangkok / 
                  </div>
                </li>,
                <li>
                  <a
                    className="link"
                    href="#"
                  >
                    Chinese New Year
                  </a>
                </li>,
                <li>
                  <a
                    className="link"
                    href="#"
                  >
                    Date Night
                  </a>
                </li>,
                <li>
                  <a
                    className="link"
                    href="#"
                  >
                    Group Dinner
                  </a>
                </li>,
                <li>
                  <a
                    className="link"
                    href="#"
                  >
                    Buisness Meals
                  </a>
                </li>,
              ],
              "className": "menu",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="legend"
                    href="#"
                  >
                    Bangkok / 
                  </div>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Bangkok / ",
                    "className": "legend",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": "Bangkok / ",
                  "type": "div",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="link"
                    href="#"
                  >
                    Chinese New Year
                  </a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Chinese New Year",
                    "className": "link",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": "Chinese New Year",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="link"
                    href="#"
                  >
                    Date Night
                  </a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Date Night",
                    "className": "link",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": "Date Night",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="link"
                    href="#"
                  >
                    Group Dinner
                  </a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Group Dinner",
                    "className": "link",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": "Group Dinner",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="link"
                    href="#"
                  >
                    Buisness Meals
                  </a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Buisness Meals",
                    "className": "link",
                    "href": "#",
                  },
                  "ref": null,
                  "rendered": "Buisness Meals",
                  "type": "a",
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span
                  className="legend"
                >
                  Search:
                </span>,
                <a
                  className="link"
                  href="#"
                >
                  Restaurant/Cuisine/Area
                </a>,
              ],
              "className": "search",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Search:",
                  "className": "legend",
                },
                "ref": null,
                "rendered": "Search:",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Restaurant/Cuisine/Area",
                  "className": "link",
                  "href": "#",
                },
                "ref": null,
                "rendered": "Restaurant/Cuisine/Area",
                "type": "a",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
